examples/workers/workers.bal

```diff
@@ -9,23 +9,24 @@ public function main() {
 
     // This block belongs to the `w1` worker.
     worker w1 {
-        http:Client httpClient = new ("https://api.mathjs.org");
-        var response = httpClient->get("/v4/?expr=2*3");
+        http:Client httpClient = new("https://api.mathjs.org");
+        var response = httpClient -> get("/v4/?expr=2*3");
         if response is http:Response {
             io:println("Worker 1 response: ", response.getTextPayload());
         }
     }
 
     worker w2 {
-        http:Client httpClient = new ("https://api.mathjs.org");
-        var response = httpClient->get("/v4/?expr=5*7");
+        http:Client httpClient = new("https://api.mathjs.org");
+        var response = httpClient -> get("/v4/?expr=5*7");
         if response is http:Response {
             io:println("Worker 2 response: ", response.getTextPayload());
         }
     }
 
     // Waits for both workers to finish.
-    _ = wait {w1, w2};
+    _ = wait { w1, w2 };
```

Comment:
This too.

Reply:
Is this an issue in the formatter?

Reply:
For a mapping constructor (and I assume thereby this also) the guideline was to not have spaces after `{` and before `}`.

https://ballerina.io/learn/coding-conventions/expressions/#record-literal